adaptive_breakpoints/lib/src/adaptive_breakpoints.dart

```diff
@@ -1,6 +1,7 @@
 // Copyright 2020, the Flutter project authors. Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
+import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
```

Comment:
This is a circular import, I don't think you need it

Reply:
Done